src/Control/Comonad/Cofree.purs

```diff
@@ -32,31 +35,30 @@ import Data.Tuple (Tuple(..))
 -- |
 -- | The `Comonad` instance supports _redecoration_, recomputing
 -- | labels from the local context.
-data Cofree f a = Cofree a (Lazy (f (Cofree f a)))
+newtype Cofree f a = Cofree (Lazy (Tuple a (f (Cofree f a))))
+
+-- | Lazily creates a value of type `Cofree f a` from a label and a
+-- | functor-full of "subtrees".
+liftC :: forall f a. (Unit -> Tuple a (f (Cofree f a))) -> Cofree f a
```

Comment:
This seems like a slightly unclear choice of name when we already have `mkCofree`. I would prefer to be more explicit and call it something like `mkCofreeLazy` but I'm open to other ideas.

Reply:
Yeah, I was just being consistent with `liftF`, which is an equally unclear name. Happy to change it to whatever.

Reply:
I agree we should change both. My open PR calls it `liftFree` actually, and moves it into a class.